packages/ERTP/test/unitTests/mathHelpers/test-natMathHelpers.js

```diff
@@ -1,58 +1,46 @@
+// @ts-check
+
+// eslint-disable-next-line import/no-extraneous-dependencies
 import test from 'ava';
+import { Far } from '@agoric/marshal';
 
-import { makeAmountMath, MathKind } from '../../../src';
+import { amountMath as m, MathKind } from '../../../src';
+import { mockBrand } from './mockBrand';
 
 // The "unit tests" for MathHelpers actually make the calls through
 // AmountMath so that we can test that any duplication is handled
 // correctly.
 
-const mockBrand = harden({
-  isMyIssuer: () => false,
-  getAllegedName: () => 'mock',
-});
-
-const amountMath = makeAmountMath(mockBrand, MathKind.NAT);
-
-test('natMathHelpers', t => {
-  const {
-    getBrand,
-    getAmountMathKind,
-    make,
-    coerce,
-    getValue,
-    getEmpty,
-    isEmpty,
-    isGTE,
-    isEqual,
-    add,
-    subtract,
-  } = amountMath;
-
-  // getBrand
-  t.deepEqual(getBrand(), mockBrand, 'brand is brand');
-
-  // getAmountMathKind
-  t.deepEqual(getAmountMathKind(), MathKind.NAT, 'amountMathKind is nat');
-
-  // make
-  t.deepEqual(make(4), { brand: mockBrand, value: 4n });
+test('natMathHelpers make', t => {
+  t.deepEqual(m.make(4n, mockBrand), { brand: mockBrand, value: 4n });
   t.throws(
-    () => make('abc'),
+    () => m.make('abc', mockBrand),
     {
-      instanceOf: TypeError,
-      message: 'abc is a string but must be a bigint or a number',
+      message: 'value (a string) must be a Nat or an array',
     },
     `'abc' is not a nat`,
   );
   t.throws(
-    () => make(-1),
-    { instanceOf: RangeError, message: '-1 is negative' },
+    () => m.make(-1, mockBrand),
+    { message: 'value (a number) must be a Nat or an array' },
     `- 1 is not a valid Nat`,
   );
+});
+
+test('natMathHelpers make no brand', t => {
+  t.throws(
+    () => m.make(4n),
+    {
+      message:
+        "The brand (an undefined) doesn't look like a brand. Did you pass a value rather than an amount?",
```

Comment:
I think "Did you pass a value rather than an amount?" is the wrong diagnostic here. `make` expects two args, not an amount.